problem_builder/public/css/problem-builder.css

```diff
@@ -0,0 +1,7 @@
+.pb-clarification span.clarification i {
+    font-style: normal;
+}
+
+.pb-clarification span.clarification i:hover {
+    color: rgb(0, 159, 230);
+}
```

Comment:
@bradenmacdonald That's a good idea, I moved the CSS rules into `mentoring.css` and renamed it to `problem_builder.css`.